bindings/python/Cargo.toml

```diff
@@ -14,8 +14,8 @@ serde = { version = "1.0", features = [ "rc", "derive" ]}
 serde_json = "1.0"
 libc = "0.2"
 env_logger = "0.7.1"
-pyo3 = "0.12"
-numpy = "0.12"
+pyo3 = "0.15.0"
+numpy = "0.15.0"
```

Comment:
numpy does not need to move in sync with pyo3; there also does not [exist](https://github.com/numpy/numpy/tags?after=v1.0) a numpy 0.15...

Reply:
It needs to. And it's [rust-numpy](https://github.com/PyO3/rust-numpy).

Reply:
OK, makes sense, thanks.